edweiss-firebase/functions/src/functions/lectures/updateQuestion.ts

```diff
@@ -0,0 +1,28 @@
+import LectureDisplay from 'model/lectures/lectureDoc';
+import { onAuthentifiedCall } from 'utils/firebase';
+import { CollectionOf, getDocumentAndRef } from 'utils/firestore';
+import { fail, ok } from 'utils/status';
+
+export const updateTodo = onAuthentifiedCall(LectureDisplay.Functions.updateQuestion, async (userId, args) => {
```

Comment:
I think you missed renaming `updateTodo` to `updateQuestion` when exporting it. This will be an issue when deploying the functions.

Reply:
Yes I had already done this correction thanks, I just didn't commit it before you reviewed